tempesta_fw/sock_clnt.c

```diff
@@ -37,15 +37,17 @@
  * ------------------------------------------------------------------------
  */
 
+unsigned short tfw_block_action_flags = TFW_BLOCK_ACTION_ERROR_REPLY;
```

Comment:
I'm wondering if we can define multiple rules

    block_action error reply;
    block_action attack drop nolog;

as described in https://github.com/tempesta-tech/tempesta/issues/658#issuecomment-270217864 using only this one variable? It seems the requirement is missed.

Reply:
The are 4 least significant bits to set/unset - so we have 16 possible states. And from the configuration statements:
> block_action error reply;
> block_action attack drop nolog;
we also have 16 possible combinations - so this should be enough.
If we have third directive. e.g.:
> block_action error reply;
then the corresponding flag bits will be rewritten to new values (if I understand the task correctly).